internal/provider/gql_generated.go

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+//
+// Code generated by gen.go; DO NOT EDIT.
+
+package provider
+
+type ClientIntegrationConfigurationOptions struct {
+	AzureConfigurationOptions                  AzureConfigurationOptions                  `graphql:"... on AzureConfigurationOptions"`
```

Comment:
should the auto-generator sort the struct for better readability?

Reply:
Yes! Good call.